sztp-agent/pkg/secureagent/utils.go

```diff
@@ -114,8 +139,9 @@ func (a *Agent) doTLSRequest(input string, url string, empty bool) (*BootstrapSe
 		}
 		log.Println(postResponse)
 	}
-	if res.StatusCode != http.StatusOK {
-		return nil, errors.New("[ERROR] Status code received: " + strconv.Itoa(res.StatusCode) + " ...but status code expected: " + strconv.Itoa(http.StatusOK))
+
```

Comment:
here, the res came with status code 204 for me
like the request is successful but no content inside
should I revert this back?

Reply:
try the existing code, I think for me it is returning also 204...

Reply:
it was looping through continuously and the agent didn't exit
previously since there was no loop, it just exited.

Reply:
![Screenshot from 2024-06-25 11-18-28](https://github.com/opiproject/sztp/assets/82646284/ebd48c61-285e-47ad-99a9-85d27b5265ef)
it will continuously execute the bootstrap sequence until status 200 comes

Reply:
i reverted it back